scheduler/duration_based_host_allocator.go

```diff
@@ -492,10 +494,37 @@ func (self *DurationBasedHostAllocator) numNewHostsForDistro(
 		grip.Error(err)
 		return 0, nil
 	}
+
+	underWaterAlert := message.Fields{
+		"distro":    distro.Id,
+		"runtime":   scheduledTasksDuration + runningTasksDuration,
+		"runner":    RunnerName,
+		"message":   "distro underwater",
+		"num_hosts": existingDistroHosts,
+	}
+
 	if !can {
+		underWaterAlert["action"] = []string{
+			"reduce workload;",
+			"add additional hosts to pool;",
+			"deactivate tasks;",
+		}
+		grip.AlertWhen(time.Duration(distroData.totalTasksDuration/float64(distro.PoolSize)) > 7*24*time.Hour,
```

Comment:
Let's put the value in a constant. Also this seems high - I feel like we want to alert well before hitting a week makespan per host

Reply:
this is (I hope/suspect) for an entire host, not for a specific host. 

and for static hosts, many are underwater and under-provisioned to the point that I clear out tasks older than a week (on the theory that people don't care about them at that point.)

I'd like to be able to have an SLA of a <1 day for any task, but that doesn't seem realistic at this point.